observability/monitoring/reviewing-monitoring-dashboards.adoc

```diff
@@ -6,40 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-ifndef::openshift-dedicated,openshift-rosa[]
-{product-title} {product-version} provides a comprehensive set of monitoring dashboards that help you understand the state of cluster components and user-defined workloads.
-endif::openshift-dedicated,openshift-rosa[]
-ifdef::openshift-dedicated,openshift-rosa[]
-{product-title} provides monitoring dashboards that help you understand the state of user-defined projects.
-endif::openshift-dedicated,openshift-rosa[]
-
-Use the *Administrator* perspective to access dashboards for the core {product-title} components, including the following items:
-
-* API performance
-* etcd
-* Kubernetes compute resources
-* Kubernetes network resources
-* Prometheus
-* USE method dashboards relating to cluster and node performance
-* Node performance metrics
-
-.Example dashboard in the Administrator perspective
-image::monitoring-dashboard-administrator.png[]
-
-Use the *Developer* perspective to access Kubernetes compute resources dashboards that provide the following application metrics for a selected project:
-
-* CPU usage
-* Memory usage
-* Bandwidth information
-* Packet rate information
-
-.Example dashboard in the Developer perspective
-image::observe-dashboard-developer.png[]
-
-[NOTE]
-====
-In the *Developer* perspective, you can view dashboards for only one project at a time.
-====
```

Comment:
@juzhao I moved this note directly to the developer section as that seemed more appropriate.

Reply:
yes, you are right